tests/test_sync_newsletter.py

```
@@ -0,0 +1,73 @@
+        self._drafts = drafts
+        self.deleted_ids: list[str] = []
+
+    async def get_drafts(self):
+        for draft in self._drafts:
+            yield draft
+
+    async def delete_email(self, email_id: str) -> None:
+        self.deleted_ids.append(email_id)
+
+
+def draft(id: str, creation_date: str, status: str = "draft") -> dict:
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)